### PR TITLE
Fixed issues with Tensorflow version

### LINKED ISSUE #158: Keras version - workers
**Describe the bug**
'workers' argument not found for newer keras versions. Raises an error. Reported by @kjjarvis on MacOs

### PR DESCRIPTION
**Pull request by teubert**
Workers argument has been removed from keras fit function. Also, the newest version of keras requires specifying metrics for each output (instead of a single metric to be applied to all). Updated LSTM to support this. Closes #158

### CODE REVIEW

**Review by kjjarvis — APPROVED** [review]
Good fix! Tests run
